sdcm/utils/ast_utils.py

```diff
@@ -0,0 +1,83 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2025 ScyllaDB
+
+import ast
+
+# based example from https://stackoverflow.com/a/70889491/459189
```

Comment:
Please, add "NOTE:" prefix. I barely distinguished it from license part.

Reply:
move it into the class doc string